grc/CMakeLists.txt

```diff
@@ -142,7 +142,19 @@ GR_PYTHON_INSTALL(
 )
 
 GR_PYTHON_INSTALL(
-    DIRECTORY core gui converter
+    DIRECTORY core
```

Comment:
This seems fine, but do you know why we need this change?

Reply:
I'll second MB's remarks: I'm fine with the change, but wonder what's going on that requires it?

This is a CMake file, so maybe some version of CMake running on Windows can't properly handle multiple DIRECTORY arguments? Or, maybe GR_PYTHON_INSTALL isn't working correct on Windows? The GR_PYTHON_INSTALL code is pretty generic CMake scripting; should work in general if it does on any OS (which I'm thinking it does on at least Linux and OSX, since all of the desired install files seem seem to be being installed).

Reply:
The cmake portion of this command does work as expected... it results in a single line command to python.exe with each file as a parameter.  (confirmed in the MSVC project files)

What doesn't work is that that command ends up exceeding the windows 8191 character limit so it gets truncated when passed to python.   So therefore some files don't get copied and things go downhill from there.

Reply:
@gnieboer OK then this change makes a LOT of sense, and is a good idea regardless of OS used. Thanks for the explanation!